### PR TITLE
Update for Minecraft 1.19

### PR DESCRIPTION
**Pull request by gniftygnome**
This update is a little beefier than the 1.18.2 update was.  I had to remove some (no longer relevant?) error handling from `PanoramaResourceReloader`, which could use a quick review.  However, it has worked fine in my limited testing.

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -5,7 +5,7 @@ archive_name=vistas
 
 minecraft_version=1.19
 yarn_mappings=1.19+build.4
-loader_version=0.14.7
```
Comment (osfanbuff63) [inline gradle.properties]: Woops on my part there, I forgot to check for a new loader version :P

**Comment by osfanbuff63** [comment]
Currently broken on 1.19.2. https://gist.github.com/osfanbuff63/4b8dbdfd51fd68b6e2a0d6ebfc4b0b0e

**Comment by gniftygnome** [comment]
> Currently broken on 1.19.2.

It just needed a rebuild with newer dep versions.  Should be fixed by the latest commit.

**Comment by osfanbuff63** [comment]
> > Currently broken on 1.19.2.
> 
> It just needed a rebuild with newer dep versions.  Should be fixed by the latest commit.

Should have tried that - I'll try a build against that commit.

**Pull request merged**